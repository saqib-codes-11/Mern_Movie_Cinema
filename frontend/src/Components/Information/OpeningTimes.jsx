import '../../CSS/Pages.css';
import { Container, Table, Jumbotron } from 'react-bootstrap';
const OpeningTimes = () => {
    return (
        <div class="background">
            <div class="fullScreen">
                <h1 class='landing-text'>Opening Times</h1>
                <Container >
                    <Jumbotron className="bgBlur">
                        <Table className="table-hover text-white " id="tableBorder" style={{ flex: 1, backgroundColor: '#A02626' }}>
                            <thead>
                                <tr>
                                    <th scope="col">Day of the Week</th>
                                    <th scope="col">Opening</th>
                                    <th scope="col"></th>
                                    <th scope="col">Closing</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <th scope="row">Monday</th>
                                    <td>13:00</td>
                                    <td> - </td>
                                    <td>22:00</td>
                                </tr>
                                <tr>
                                    <th scope="row">Tuesday</th>
                                    <td>13:00</td>
                                    <td> - </td>
                                    <td>22:00</td>
                                </tr>
                                <tr>
                                    <th scope="row">Wednesday</th>
                                    <td>13:00</td>
                                    <td> - </td>
                                    <td>22:00</td>
                                </tr>
                                <tr>
                                    <th scope="row">Thursday</th>
                                    <td>13:00</td>
                                    <td> - </td>
                                    <td>22:00</td>
                                </tr>
                                <tr>
                                    <th scope="row">Friday</th>
                                    <td>13:00</td>
                                    <td> - </td>
                                    <td>23:30</td>
                                </tr>
                                <tr>
                                    <th scope="row">Saturday</th>
                                    <td>9:30</td>
                                    <td> - </td>
                                    <td>23:30</td>
                                </tr>
                                <tr>
                                    <th scope="row">Sunday</th>
                                    <td>9:30</td>
                                    <td> - </td>
                                    <td>23:30</td>
                                </tr>
                            </tbody>
                        </Table>
                    </Jumbotron>
                </Container>
            </div>
        </div>
    )
}
export default OpeningTimes;