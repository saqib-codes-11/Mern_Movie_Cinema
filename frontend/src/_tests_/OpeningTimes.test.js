import React from 'react';
import OpeningTimes from '../Components/Information/OpeningTimes';
import renderer from 'react-test-renderer';

test("It matches the snapshot", () => {
    const tree = renderer.create(<OpeningTimes />).toJSON();
    expect(tree).toMatchSnapshot();

})