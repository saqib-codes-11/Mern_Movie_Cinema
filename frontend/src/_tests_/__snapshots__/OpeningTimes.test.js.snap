// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches the snapshot 1`] = `
<div
  class="background"
>
  <div
    class="fullScreen"
  >
    <h1
      class="landing-text"
    >
      Opening Times
    </h1>
    <div
      className="container"
    >
      <div
        className="bgBlur jumbotron"
      >
        <table
          className="table-hover text-white  table"
          id="tableBorder"
          style={
            Object {
              "backgroundColor": "#A02626",
              "flex": 1,
            }
          }
        >
          <thead>
            <tr>
              <th
                scope="col"
              >
                Day of the Week
              </th>
              <th
                scope="col"
              >
                Opening
              </th>
              <th
                scope="col"
              />
              <th
                scope="col"
              >
                Closing
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <th
                scope="row"
              >
                Monday
              </th>
              <td>
                13:00
              </td>
              <td>
                 - 
              </td>
              <td>
                22:00
              </td>
            </tr>
            <tr>
              <th
                scope="row"
              >
                Tuesday
              </th>
              <td>
                13:00
              </td>
              <td>
                 - 
              </td>
              <td>
                22:00
              </td>
            </tr>
            <tr>
              <th
                scope="row"
              >
                Wednesday
              </th>
              <td>
                13:00
              </td>
              <td>
                 - 
              </td>
              <td>
                22:00
              </td>
            </tr>
            <tr>
              <th
                scope="row"
              >
                Thursday
              </th>
              <td>
                13:00
              </td>
              <td>
                 - 
              </td>
              <td>
                22:00
              </td>
            </tr>
            <tr>
              <th
                scope="row"
              >
                Friday
              </th>
              <td>
                13:00
              </td>
              <td>
                 - 
              </td>
              <td>
                23:30
              </td>
            </tr>
            <tr>
              <th
                scope="row"
              >
                Saturday
              </th>
              <td>
                9:30
              </td>
              <td>
                 - 
              </td>
              <td>
                23:30
              </td>
            </tr>
            <tr>
              <th
                scope="row"
              >
                Sunday
              </th>
              <td>
                9:30
              </td>
              <td>
                 - 
              </td>
              <td>
                23:30
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</div>
`;
